ci-operator/config/openshift/installer/openshift-installer-master.yaml

```diff
@@ -207,6 +207,7 @@ tests:
     cluster_profile: gcp
     workflow: openshift-e2e-gcp
 - as: e2e-alibaba
+  optional: true
```

Comment:
Might also want to add this?
skip_if_only_changed: (^docs/)|((^|/)OWNERS(_ALIASES)?$)|((^|/)[A-Z]+\.md$)|(^(data/.*|pkg/.*|images)/(aws|azure|baremetal|gcp|libvirt|openstack|ovirt|vsphere)/)

Reply:
@patrickdillon I think this would be great ^^